README.rst

```diff
@@ -257,6 +257,100 @@ commands:
 ``galayxctl follow``                ``journalctl -f -u 'galaxy-*'``
 =================================== ==================================================================
 
+Zero-Downtime Restarts
+----------------------
+
+Prior to Gravity 1.0, the preferred solution for performing zero-downtime restarts was `unicornherder`_. However, due to
+limitations in the unicornherder software, it does not always successfully perform zero-downtime restarts. Because of
+this, Gravity is now able to perform rolling restarts of gunicorn services if more than one gunicorn is configured.
+
+To run multiple gunicorn processes, configure the ``gunicorn`` section of the Gravity configuration as a *list*. Each
+item in the list is a gunicorn configuration, and can have all of the same parameters as a single gunicorn
+configuration:
+
+.. code:: yaml
+
+    gravity:
+      gunicorn:
+        - bind: unix:/srv/galaxy/var/gunicorn0.sock
+          workers: 4
+        - bind: unix:/srv/galaxy/var/gunicorn1.sock
+          workers: 4
+
+.. caution::
+
+   This will start multiple Galaxy servers with the same ``server_name``. If you have not configured separate Galaxy
+   processes to act as job handlers, your gunicorn processes will handle them, resulting in job errors due to handling
+   the same job multiple times. See the Gravity and Galaxy documentation on configuring handlers.
+
+Your proxy server can balance load between the two gunicorns. For example, with nginx:
+
+.. code:: nginx
+
+    upstream galaxy {
+        server unix:/srv/galaxy/var/gunicorn0.sock;
+        server unix:/srv/galaxy/var/gunicorn1.sock;
+    }
+
+    http {
+        location / {
+            proxy_pass http://galaxy;
+        }
+    }
+
+To perform the readiness check on a gunicorn using a UNIX domain socket as in this example, the `requests-unixsocket`_
+Python library must be installed. This is not installed by default so as not to add an additional dependency to Galaxy,
+but you can install it when installing Gravity with:
+
+.. code:: console
+
+    $ pip install 'gravity[unixsocket]'
```

Comment:
This is a single dependency that doesn't seem to bring any extra dependencies, I think adding this as a requirement is totally fine and then you don't need to worry about https://github.com/galaxyproject/gravity/pull/81/files#r1006508902

Reply:
yeah that sounds better :)

Reply:
Works for me, add another to the pile.